tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/colorization_evaluator.py

```diff
@@ -439,6 +445,32 @@ def set_input_and_output(self):
             self.with_prefix = with_prefix
 
 
+class ColorizationTestModelPytorch(ColorizationTestModel):
```

Comment:
it is duplicate from #1669 . Do you plan merge this request before this or put 2 models in one request?

Reply:
BTW, I do not see any adoptions to work with multiple input in evaluator. Do you plan to do it?

Reply:
1) I plan merge #1669 before this.
2) This model is able to work in 2 modes: without user points ( with 1 input similar to other colorization models) and with user hints (multiple inputs). Accuracy is measured without user input, so we can use `colorization_evaluator` without any changes.